docs/reference/versions/index.md

```diff
@@ -1,35 +1,26 @@
 # Service version matrix
 
-Services in {{brand}} are updated on a regular basis and
-on a rolling schedule.
+Services in {{brand}} are updated on a regular basis and on a rolling schedule.
 
-This section lists the cloud API service versions available in each
-{{brand}} region.
+vThis section lists the cloud API service versions available in each {{brand}} region.
```

Comment:
There's an extra "v" at the beginning of the sentence:
```suggestion
This section lists the cloud API service versions available in each {{brand}} region.
```

Reply:
Oh, weird. Fixed now; no idea how that got in. Well spotted, thanks!